zebrad/src/components/inbound.rs

```diff
@@ -19,7 +19,7 @@ use futures::{
     stream::Stream,
 };
 use tokio::sync::oneshot::{self, error::TryRecvError};
-use tower::{buffer::Buffer, timeout::Timeout, util::BoxService, Service, ServiceExt};
+use tower::{buffer::Buffer, timeout::Timeout, util::BoxService, BoxError, Service, ServiceExt};
```

Comment:
Suggestion (2/4): Can we use `zebrad::BoxError` instead?

Reply:
Done.